src/aui/tabart.cpp

```diff
@@ -1522,6 +1531,10 @@ wxSize wxAuiSimpleTabArt::GetTabSize(wxDC& dc, wxWindow* WXUNUSED(wnd), const wx
     wxCoord tabHeight = measuredTextY + 4;
     wxCoord tabWidth = measuredTextX + tabHeight + 5;
 
+    if (bitmap.IsOk()) {
+        tabWidth += bitmap.GetWidth() + 6;
```

Comment:
You are right. I have given a look on the management of the customizing of the size of the tabs and it is not supported at all. The little bit that is implemented does not work. I think that deserves a separate thread to treat the subject seriously.